src/configure-nilrt-snac

```diff
@@ -127,7 +129,7 @@ install_cryptsetup() {
 # Rips niauth out of the system.
 remove_niauth() {
 	log INFO Removing NIAuth...
-	
+
```

Comment:
Nit: We should be consistent about spacing. Methods above such as `configure_wireguard` don't have a blank line after the log INFO line.

Reply:
I agree that we should be consistent, but I don't want to make any additional changes of the existing functions in this pull request.

I'm not familiar with any shell script linters.